src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/UserEntity.kt

```diff
@@ -386,6 +386,7 @@ enum class UserRole(val service: ServiceName, val cas1ApiValue: ApprovedPremises
       UserPermission.CAS1_OUT_OF_SERVICE_BED_CREATE,
       UserPermission.CAS1_SPACE_BOOKING_LIST,
       UserPermission.CAS1_SPACE_BOOKING_RECORD_ARRIVAL,
+      UserPermission.CAS1_SPACE_BOOKING_RECORD_DEPARTURE,
```

Comment:
can we add this to cru member and janitor as well please (going forward i think janitor needs everything new adding to it). See https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4357851312/User+roles+in+Approved+Premises